tests/step_implementers/deploy/test_argocd.py

```diff
@@ -1366,7 +1366,7 @@ def test_ArgoCD__clone_repo_success_new_branch(self, git_mock):
     @patch.object(sh, 'git')
     def test_ArgoCD__clone_repo_success_existing_branch(self, git_mock):
         repo_dir = '/does/not/matter'
-        repo_url = 'git@git.ploigos.xyz:/foo/test.git'
+        repo_url = 'https://{username}:{password}@git.ploigos.xys/foo'
```

Comment:
rather then changing these tests, can you rename them *_ssh and then coopy them and name them `*_https` with these changes?